lib/modules/platform/bitbucket-server/index.md

```diff
@@ -59,7 +59,7 @@ export RENOVATE_GIT_FS=http
 export RENOVATE_USERNAME=renovate-bot
 export RENOVATE_PASSWORD=abc123456789!
 export LOG_LEVEL=debug
-yarn start --autodiscover=true
+pnpm start --autodiscover=true
```

Comment:
Are you sure about this change? This is a user-facing document.

Reply:
```suggestion
npx renovate --autodiscover=true
```